src/formatMarkup.js

```diff
@@ -54,11 +54,17 @@ export const diffableFormatter = function (markup, options) {
   if (typeof(options.selfClosingTag) !== 'boolean') {
     options.selfClosingTag = false;
   }
+  if (!Array.isArray(options.tagsWithWhitespacePreserved) && 
+      typeof(options.tagsWithWhitespacePreserved) !== 'boolean') {
```

Comment:
boooooo bad prettier
```suggestion
  if (
    !Array.isArray(options.tagsWithWhitespacePreserved) && 
    typeof(options.tagsWithWhitespacePreserved) !== 'boolean'
  ) {
```